### ISSUE TITLE
Build spawned on PR create, but not on new commits

### ISSUE THREAD

**Issue by davidmichaelkarr** [KyleLNicholls/parameterized-builds#191]
We have BitBucket v5.10.1.  I don't know what version of the plugin we have (I don't have access to system admin settings, just repo settings).

We configured this for one repo, and it successfully spawned a build when we created the PR.  However, when we pushed additional changes to the branch, nothing happened.  In the Triggers list, we have MANUAL, PR_OPENED, PR_REOPENED, and PR_SOURCE_RESCOPED.  We found that triggering it manually also worked.  It just doesn't seem to handle PR_SOURCE_RESCOPED.

**Comment by a-hacker** [comment]
Hi @davidmichaelkarr. Thank you for submitting this bug report. Unfortunately, we are no longer supporting Bitbucket Server 5.X now that Bitbucket Server 6 has released. 

I'll still look into this though and if I can find a fix, I can create a custom release jar. This release would not be pushed to the marketplace so your bitbucket admin would need to manually install the updated jar. Does that sound reasonable?

**Comment by davidmichaelkarr** [comment]
I don't know.  I don't suppose I have a choice. :)

**Comment by a-hacker** [comment]
Hey @davidmichaelkarr. I just tested our app version 3.2.10 on bitbucket server 5.4.0 and the PR_SOURCE_RESCOPED trigger worked as expected. 3.2.10 is the only version of the the app with the PR_SOURCE_RESCOPED trigger available on Bitbucket Server 5.X. 

Could you see if there are any errors reported in your Bitbucket logs when a change is pushed to a branch with a pull request?

**Comment by davidmichaelkarr** [comment]
What I realized after creating this ticket is that this is not a problem
with the plugin handling the event, it's a problem with BitBucket firing
the event.  We still have the Pull Request Notification plugin enabled for
our repositories, and I realized that we weren't handling this event in the
PRN either.  In fact, I've seen this happen multiple times before. It just
stops generating it one day, and then it works again the next morning. This
morning it appears to be working again, because my test is showing both the
PRN and the PBJ plugin is handling and processing the event.
